README.md

```diff
@@ -14,6 +12,21 @@ A mongodb database named lizardboard must be created prior to starting the appli
 - Ensure `mongo` is running
 - yarn install
 
+## HTTP API
+description                                  | CRUD   | verb   | path                                     |
+-------------------------------------------- | ------ | ------ | ---------------------------------------- |
+Show all users                               | read   | get    | /api/v1/users                            |
```

Comment:
Kind of nitpicky, but the column title says CRUD, but then entries are listed in RC order here.  Can you please order entries in this table consistently (in order CRUD)?